go/client/cmd_wait.go

```diff
@@ -46,39 +51,36 @@ func (c *CmdWait) ParseArgv(ctx *cli.Context) error {
 	if c.duration.Seconds() <= 0 || c.duration.Seconds() > maxWaitTime.Seconds() {
 		return fmt.Errorf("invalid duration %s, must be between 1s and %s", c.duration, maxWaitTime)
 	}
+
+	c.includeKBFS = ctx.Bool("include-kbfs")
+
 	return nil
 }
 
 func (c *CmdWait) Run() error {
-	ctx, cancel := context.WithTimeout(context.Background(), c.duration)
+	var timeoutDuration time.Duration
+	if c.includeKBFS {
+		timeoutDuration = c.duration * 2
```

Comment:
@heronhaye is there a way to reset the timeout time, or is this ok?

Reply:
`context.WithTimeout` is for _fatal_ timeouts, I'd say. My suggestion would be to have a `WithCancel` and some logic to trigger the cancel function _if_ the timeout passes.

Reply:
I'd probably make another context.WithTimeout from background for the KBFS check. Also add a comment in --include-kbfs that it'll wait an additional N seconds for kbfs to start up if it's passed

Reply:
Cool it works out